injected/src/content-feature.js

```diff
@@ -311,23 +194,15 @@ export default class ContentFeature {
      */
     callLoad(args) {
         const mark = this.monitor.mark(this.name + 'CallLoad');
-        this.#args = args;
+        this.args = args;
         this.platform = args.platform;
-        this.#bundledConfig = args.bundledConfig;
-        // If we have a bundled config, treat it as a regular config
-        // This will be overriden by the remote config if it is available
-        if (this.#bundledConfig && this.#args) {
-            const enabledFeatures = computeEnabledFeatures(args.bundledConfig, args.site.domain, this.platform.version);
-            this.#args.featureSettings = parseFeatureSettings(args.bundledConfig, enabledFeatures);
-        }
-        this.#trackerLookup = args.trackerLookup;
-        this.#documentOriginIsTracker = args.documentOriginIsTracker;
+        this.initLoadArgs(args);
```

Comment:
This initLoadArgs becomes part of the constructor (just calling super with the args).